addons/mail/models/mail_message.py

```diff
@@ -905,9 +902,12 @@ def _message_format(self, fnames, format_reply=True, legacy=False):
                 'pinned_at': message_sudo.pinned_at,
                 'record_name': record_name,
             })
-
         return vals_list
 
+    def _message_format_extras(self, format_reply):
+        self.ensure_one()
+        return {}
```

Comment:
not happy with this, make _messages_format and _message_format ??

Reply:
I'll leave it like this for now, it will be gone when we split tables.

Reply:
Won't we need the format for notifications anyway even in mail ?

Reply:
Yes we'll probably need a formatter, but not the `format_reply` param which is channel only, and it probably won't even be the same format between mail and discuss when they become 2 different message tables.